### PR TITLE
verilator: update to 4.106

### PR DESCRIPTION
**Pull request by umarcor**

### CODE REVIEW

File: mingw-w64-verilator/PKGBUILD [hunk 1]
```diff
@@ -10,12 +12,18 @@ license=("LGPL")
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "flex")
 depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs")
 
-source=("https://www.veripool.org/ftp/${_realname}-${pkgver}.tgz")
-sha256sums=('3a60b7b8eb5a19f3becb6184bdd2ce8919fb7c049441dfce6fa9cc7bf88379d0')
+source=("${_realname}::https://codeload.github.com/${_realname}/${_realname}/tar.gz/v${pkgver}")
```
Comment (lazka) [inline mingw-w64-verilator/PKGBUILD:15]: you need to add the patch to source

File: mingw-w64-verilator/PKGBUILD [hunk 2]
```diff
@@ -10,12 +12,18 @@ license=("LGPL")
...
+sha256sums=('023327b30b68268c6cbf2debf8e8587ea5340b1b0e8b119776a3e7a9e5124140')
 
-build() {
+prepare() {
   [[ -d "${srcdir}/build-${MINGW_CHOST}" ]] && rm -rf "${srcdir}/build-${MINGW_CHOST}"
-  cp -rf "${srcdir}/verilator-${pkgver}" "${srcdir}/build-${MINGW_CHOST}"
+  cp -rf "${srcdir}/${_realname}-${pkgver}" "${srcdir}/build-${MINGW_CHOST}"
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  sh autoconf
+  patch -p1 -i "${srcdir}"/../0002-define-WEXITSTATUS.patch
```
Comment (lazka) [inline mingw-w64-verilator/PKGBUILD:23]: "${srcdir}"/0002-define-WEXITSTATUS.patch

File: mingw-w64-verilator/PKGBUILD (hunk 1 above)
Comment (umarcor) [reply mingw-w64-verilator/PKGBUILD]: Done.

File: mingw-w64-verilator/PKGBUILD (hunk 2 above)
Comment (umarcor) [reply mingw-w64-verilator/PKGBUILD]: Done, both.

**Pull request merged**